code/game/objects/items/tools/crowbar.dm

```diff
@@ -1,5 +1,5 @@
 /obj/item/crowbar
-	name = "pocket crowbar"
+	name = "crowbar"
 	desc = "A small crowbar. This handy tool is useful for lots of things, such as prying floor tiles or opening unpowered doors."
```

Comment:
```suggestion
	desc = "This handy tool is useful for lots of things, such as prying floor tiles or opening unpowered doors."
```
Guess we can remove this since its no longer a small crowbar.